site/app/views/course/CourseMaterialsView.php

```diff
@@ -26,7 +27,7 @@ public function listCourseMaterials(array $course_materials_db) {
         $folder_ids = [];
         $links = [];
         $base_view_url = $this->core->buildCourseUrl(['course_material']);
-
+        $begining_of_time_date = DateUtils::BEGINING_OF_TIME;
```

Comment:
Looks good to me.  One minor nitpick you don't necessarily have to change: since you're just using this variable in one place, it might be better to just inline it below.